spec/amaysim_logger/rails_controller_helper_spec.rb

```diff
@@ -33,7 +33,7 @@ class TestController < ActionController::Base
         # rubocop:disable RSpec/ExampleLength
         it 'logs the http request with a generated correlation id' do
           log = {
-            msg: 'controller request',
+            msg: 'API request',
```

Comment:
As a controller helper I'm not sure if we want to change it to `API`. Non api controllers should be able to use this helper as well.